### PR TITLE
lance-usopen-branch

### PR DESCRIPTION
**Pull request by LanceAndersen**
/reviewer  add @JoeWang-Java

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Reviewers
 * joehw - **Reviewer** ⚠️ Added manually

### Download
`$ git fetch https://git.openjdk.java.net/playground pull/53/head:pull/53`
`$ git checkout pull/53`

### CODE REVIEW

File: MissingNYC2020USOpen.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Missing being in NYC for the 2020 US Open
```
Comment (JoeWang-Java) [inline MissingNYC2020USOpen.txt:1]: uh no, no missing games :-)

what happens if I hit the 'start a review' button for this file?

File: src/java.sql/share/classes/java/sql/Statement.java [hunk 2]
```diff
@@ -32,7 +32,7 @@
  * <P>The object used for executing a static SQL statement
  * and returning the results it produces.
  * <P>
- * By default, only one <code>ResultSet</code> object per <code>Statement</code>
+ * By default, only one {@code ResultSe} object per <code>Statement</code>
```
Comment (JoeWang-Java) [inline src/java.sql/share/classes/java/sql/Statement.java:35]: Now, after having started review, that button no longer show, can only add comment

Comment (JoeWang-Java) [reply src/java.sql/share/classes/java/sql/Statement.java:35]: I see I can "finish your review" (2)

Can you, the author, do the same?

**Comment by JoeWang-Java** [comment]
and I'm going to try the "request changes" option, if you don't mind ;-)

**Review by JoeWang-Java — CHANGES_REQUESTED** [review]

File: src/java.sql/share/classes/java/sql/Statement.java (hunk 2 above)
Comment (LanceAndersen) [reply src/java.sql/share/classes/java/sql/Statement.java:35]: Looking now

**Review by LanceAndersen — COMMENTED** [review]
rplied with a comment

**Comment by JoeWang-Java** [comment]
I don't see any effect, I can still select any of the three options: comment, approve, request changes

**Comment by JoeWang-Java** [comment]
I wonder if it'd allow you to 'integrate'...

**Pull request closed without merging**